[2021-12-18 09:03:49,610] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.cd_home 2021-12-18T09:03:37.133567+00:00 [queued]>
[2021-12-18 09:03:49,628] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.cd_home 2021-12-18T09:03:37.133567+00:00 [queued]>
[2021-12-18 09:03:49,628] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 09:03:49,628] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-18 09:03:49,628] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 09:03:49,650] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): cd_home> on 2021-12-18T09:03:37.133567+00:00
[2021-12-18 09:03:49,652] {standard_task_runner.py:52} INFO - Started process 83400 to run task
[2021-12-18 09:03:49,664] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'cd_home', '2021-12-18T09:03:37.133567+00:00', '--job-id', '48', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmpl5m0ja2q', '--error-file', '/tmp/tmpxddhxjn7']
[2021-12-18 09:03:49,666] {standard_task_runner.py:77} INFO - Job 48: Subtask cd_home
[2021-12-18 09:03:49,735] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.cd_home 2021-12-18T09:03:37.133567+00:00 [running]> on host airflowvm
[2021-12-18 09:03:49,910] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=cd_home
AIRFLOW_CTX_EXECUTION_DATE=2021-12-18T09:03:37.133567+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-18T09:03:37.133567+00:00
[2021-12-18 09:03:49,910] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-18 09:03:49,910] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'cd /home/airflow']
[2021-12-18 09:03:49,924] {subprocess.py:75} INFO - Output:
[2021-12-18 09:03:49,926] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-18 09:03:49,963] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=cd_home, execution_date=20211218T090337, start_date=20211218T090349, end_date=20211218T090349
[2021-12-18 09:03:50,005] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-18 09:03:50,040] {local_task_job.py:151} INFO - Task exited with return code 0
